code/datums/quirks/negative_quirks/negative_quirks.dm

```diff
@@ -1535,3 +1535,20 @@
 	SIGNAL_HANDLER
 	quirk_holder.client.give_award(/datum/award/achievement/misc/debt_extinguished, quirk_holder)
 	UnregisterSignal(source, COMSIG_MOB_LOGIN)
+
+/datum/quirk/numb
+	name = "Numb"
+	desc = "You can't feel pain at all."
+	icon = FA_ICON_STAR_OF_LIFE
+	value = -4
+	gain_text = "You feel your body becoming numb."
+	lose_text = "The numbness subsides."
+	medical_record_text = "Patient's body has poorly developed nervous system leading to complete inability to feel pain. "
```

Comment:
hate to bikeshed here, but this sentence still just reads weirdly. we should rewrite it

(there's also an extra space after your period)

```suggestion
	medical_record_text = "The patient exhibits congenital hypoesthesia, making them insensitive to pain stimuli."
```